### ISSUE TITLE
Read aloud bogs down in MathML, could handle this better

### ISSUE THREAD

**Issue by ways2read** [edrlab/thorium-reader#1593]
Steps to replicate:

- open Accessibility Tests Mathematics v1.1.1 from `https://epubtest.org/books/Accessibility-Tests-Mathematics-v1.1.1.epub`
- turn on read aloud and listen to the expression in tests mathml-010 or mathml-020
Read aloud announces gibberish and then gets stuck in the expression, speech then stops, the highlight slowly crawls through the expression, and then after a few seconds speech resumes with the text following.

Ideally Thorium Reader would use MathJax to generate a text string for the read aloud. If this is not possible, desired behaviour is for the read aloud to ignore the MathML but announce any alt text if present,

**Comment by danielweck** [comment]
Hello Richard, thank you for reporting this. MathJax `aria-label` generation is now activated by default. Thorium's TTS read aloud engine has a fallback strategy on the MathML `alttext`, just in case MathJax fails to produce the label in the document language. The last-resort fallback is a readout of the raw text contents inside the MathML fragment.

**Comment by ways2read** [comment]
Tested alpha.1.1533362900 on Windows and confirmed read aloud now announces the math in MathML - very cool!

Will check out on macOS and do some stress testing also.

**Comment by danielweck** [comment]
Note that I have literally just pushed an update that TTS-reads plain MathML (i.e. MathJax not activated in Thorium). The automatic build should be ready shortly.

**Comment by danielweck** [comment]
build in progress: https://github.com/edrlab/thorium-reader/actions/runs/1535975529

**Comment by danielweck** [comment]
I've just tested https://epubtest.org/books/Accessibility-Tests-Mathematics-v1.1.1.epub in Thorium with and without MathJax, everything seems to work ... well, the expandable/collapsible summary/detail MathML is TTS-read even when collapsed so the highlight focus jumps to the top (bug to fix), but the MathML `alttext` vs. MathJax `aria-label` works well.

**Comment by danielweck** [comment]
Note that the infamous "linear algebra" sample EPUB3 contains `alttext` attributes with hard-coded "Alternative text not available" value ... so not a great publication to test with!

For example:

https://github.com/IDPF/epub3-samples/blob/0b3e80553f509fbdfe591008cd7d1e804b24db54/30/linear-algebra/EPUB/xhtml/fcla-xml-2.30li100.xhtml#L41

https://idpf.github.io/epub3-samples/30/samples.html

**Comment by danielweck** [comment]
Perhaps a follow-up issue? (regression) https://github.com/edrlab/thorium-reader/issues/1602
